pandas/_libs/pd_parser.c

```diff
@@ -0,0 +1,270 @@
+/*
+
+Copyright (c) 2023, PyData Development Team
+All rights reserved.
+
+Distributed under the terms of the BSD Simplified License.
+
+*/
+#define _PANDAS_PARSER_IMPL
+
+#include "pd_parser.h"
+#include "src/parser/io.h"
+
+static int pandas_to_double(char *item, double *p_value, char sci, char decimal,
```

Comment:
This and floatify were originally implemented in the parse_helper header, so nice to move back to the implemtnation as part of this